schema.sql

```diff
@@ -67,4 +67,15 @@ CREATE TABLE visits (
     date_of_visit DATE,
     FOREIGN KEY (animal_id) REFERENCES animals (id),
     FOREIGN KEY (vet_id) REFERENCES vets (id) 
-);
\ No newline at end of file
+);
+
+-- Run the following query to add an extra column to the owners table:
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+--indexing 
+CREATE INDEX visits_animal_id_idx ON visits(animal_id);
+
+CREATE INDEX visits_vets_id_idx ON visits(vets_id);
```

Comment:
For query : `SELECT * FROM visits where vet_id = 2`:
The screenshot with the title `BEFORE` shows an execution time of far less than 1000ms which should be around 1000ms.
Please try running the populate database query a few times more as described [here](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md#:~:text=4%20Depening%20on,bigger%20than%201000ms.)